VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+| CVE      | [Common Vulnerabilities and Exposures](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) |
+| KCSA     | Kata Containers Security Announcement                           |
+| KCSN     | Kata Containers Security Note                                   |
+| KCST     | Kata Containers Security Team                                   | 
+| PR       | [A GitHub Pull Request](https://help.github.com/articles/about-pull-requests/) |
+| VMT      | The Kata Containers Vulnerability Management Team               |
+
+# Supported versions
+
+> **FIXME** Before we can make statements about which stable versions we fix, we need to define the Kata Containers release and maintenance process.
+
+The VMT coordinates patches fixing vulnerabilities in supported stable branches of Kata Containers, in addition to the master branch (next version under development).
+
+# Process
+
+Each security bug is assigned a VMT *coordinator* (a member from the VMT) that will drive the fixing and disclosure process.
+Here are the steps we follow.
+
+![VMT process workflow](VMT-process.png)
+
+## Reception
+
+A report is received via an email to the [VMT mailing list](mailto:kata-vmt@lists.katacontainers.io).
+
+> **FIXME** We need to decide where and how reports are handled.  The obvious choice is to use a private GitHub repo alongside the other Kata Containers repos - but, we would have to pay for that (private repos are not free).  Another alternative may be to use any OSF infrastructure already in place and in use for other OSF projects, if any exists.
+> This document will be drafted under the assumption we are going to use private GitHub repos and issues, and updated later if that is not the case.
+
+The first steps performed by the VMT are to confirm the validity of the report, create a private GitHub Issue if necessary, prefix the description with an [embargo reminder](templates/reception-embargo-reminder.md), create a Kata Containers Security Announcement ([KCSA](templates/KCSA.md)) Issue, and subscribe the project's core security review team for confirmation of impact and determination of affected branches.
+Reports starting with an *Incomplete* KCSA Issue should have a corresponding [incomplete reception message](templates/reception-incomplete-message.md) added in a comment.
+Once the VMT confirms a KCSA is warranted, the KCSA Issue status will be set to *Confirmed*.
+If the need for a KCSA is challenged, the KCSA Issue status should be set back to *Incomplete* until that question is resolved.
+
+> **FIXME** Do we need to create a Kata Containers Security Team (KCST) (that is, a team of security experts in the Kata Containers community), along with the VMT (the key difference being the VMT drive the process, whereas the KCST are security experts and engineers who can make technical decisions and contributions).
+
+> **FIXME** We also need to define where we store and publish KCSN's - I expect on a GitHub md file in the appropriate repo?
```

Comment:
Perhaps in this doc somewhere until there's enough other security collateral to warrant a repo?

Reply:
I'd rather not put them in this doc. It's going to be pretty simple to have a single md file to list them in, and that could either be in one of the existing repos (I suspect https://github.com/kata-containers/documentation is more appropriate than the community repo).
And, new repos for public info are cheap - so we could just make a new 'security' repo to store that file in. I'm happy with a security file, and probably in a security sub-folder, in the docs report.